android-components/components/feature/search/src/test/java/mozilla/components/feature/search/telemetry/incontent/InContentTelemetryTest.kt

```diff
@@ -250,18 +230,18 @@ class InContentTelemetryTest {
             },
         )
 
-        telemetry.trackPartnerUrlTypeMetric(url, listOf())
+        telemetry.trackPartnerUrlTypeMetric(url, createCookieList())
 
         assertEquals(1, facts.size)
         assertEquals(Component.FEATURE_SEARCH, facts[0].component)
         assertEquals(Action.INTERACTION, facts[0].action)
         assertEquals(InContentTelemetry.IN_CONTENT_SEARCH, facts[0].item)
-        assertEquals("baidu.in-content.sap-follow-on._1000969a", facts[0].value)
+        assertEquals("bing.in-content.sap-follow-on.mozl", facts[0].value)
     }
 
     @Test
-    fun `GIVEN a Bing sap-follow-on with cookies WHEN trackPartnerUrlTypeMetric is called THEN emit an appropriate IN_CONTENT_SEARCH fact`() {
-        val url = "https://www.bing.com/search?q=aaa&pc=MOZMBA&form=QBRERANDOM"
+    fun `GIVEN a Google organic search WHEN trackPartnerUrlTypeMetric is called THEN emit an appropriate IN_CONTENT_SEARCH fact`() {
```

Comment:
Why is this test and some of the ones below it changing from Google to Bing? Is the diff being strange, or is this a real change?

Reply:
It's not actually that.It's just how Git is showing those in that way. But if you see all the test, removing Baidu caused moving the rest up and hence it looks that way